### DISCUSSION TITLE
Single PCB

### DISCUSSION THREAD

**Discussion by lazaj** [gusmanb/logicanalyzer#59] [Ideas]
Why not create a single PCB with the level shifters and a jumper to select the input voltage?

**Comment by gusmanb** [comment]
Hi.

There are two main reasons, first a lot of people don't need at all 5v, nowadays 5v is obsolete and the most common voltage is 3.3v (yes, 1.8v is also common and needs the board but is more for FPGA's and alike), so a lot of people don't need the shifter board.

Second, I have planned to add more functions to the device, main one is to convert the device into a two channel oscilloscope, and for that with the modular design you will only need to change the secondary board instead of having two completely different devices. In fact I already have all the components for the oscilloscope board but some health issues don't allow me to dedicate as much time as I would like to this so it's taking more time than what I expected...

Cheers!
